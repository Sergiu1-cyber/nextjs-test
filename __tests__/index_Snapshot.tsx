import {render} from '@testing-library/react'
import Home from '../pages'

it('Snapshot Home', () => {
  const {container} = render(<Home />)
  expect(container).toMatchSnapshot()
})

