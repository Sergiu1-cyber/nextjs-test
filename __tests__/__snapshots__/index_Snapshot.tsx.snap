// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Home 1`] = `
<div>
  <div>
    <h1
      class="text-red-500"
    >
      Home Page
    </h1>
    <img
      alt="img"
      data-nimg="1"
      decoding="async"
      height="40"
      loading="lazy"
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style="color: transparent;"
      width="40"
    />
  </div>
</div>
`;
